projects/transformers/experiments/sparse_bert.py

```diff
@@ -48,8 +48,34 @@
 )
 
 
+static_sparse_encoder_bert_100k = deepcopy(bert_100k)
+static_sparse_encoder_bert_100k.update(
+    # Model Arguments
+    overwrite_output_dir=False,
+    model_type="static_sparse_encoder_bert",
+    config_kwargs=dict(
```

Comment:
Is the plan to use the same model, but keep evolving so config takes more parameters, like to specify which layers to sparsify and which method to use for sparsification ? (eg random uniform by layer, random uniform by head, or some other form of sparsity initialization).

Reply:
We could do that. I think that will work best for minor tweaks (e.g. adjusting the sparse init). We can also make new models as needed. For instance, we can probably make a `VDropBertModel` or something like that at some point.